### ISSUE TITLE
Exception after closing last tab

### ISSUE THREAD

**Issue by cscase** [cscase/SeqTool#2]
An exception gets thrown if you close the last open tab using the context menu.

**Comment by cscase** [comment]
Same thing if you close the last tab using the File -> Close option.

**Comment by cscase** [comment]
Found that if I comment out the listener on the tabPane selection property, this doesn't happen. So, something is going on probably related to my updateInfoBox method.

**Comment by cscase** [comment]
This seems to be the line that causes it:
SeqTab currentTab = (SeqTab) tabPane.getTabs().get(currentTabIndex);

**Comment by cscase** [comment]
I found that when you close the last tab, the tab index gets set to -1, which I guess means nothing is selected. So, the line I pasted in the last comment was in effect trying to access the -1 element of an ArrayList and causing an index out of bounds exception. So, I was able to fix it by adding the following at the beginning of my updateInfoBox method:
        if (currentTabIndex == -1) {
            clearInfoBox();
            return;
        }

This also makes it so that you don't get lingering stuff in the info box after having closed the last tab - it clears it all out.
